### DISCUSSION TITLE
PS3 Eye Settings

### DISCUSSION THREAD

**Discussion by Filion91** [opentrack/opentrack#1466] [General]
So I was using a cheap webcam along with Aruco markers for head-tracking; it worked very well as I was able to adjust the contrast/brightness/saturation to get a good picture for the tracker in my terrible lighting conditions. I decided to upgrade to a PS3Eye since it was cheap and boasts an impressive FPS, however, it seems like with the driver provided with OpenTrack only gain and exposure are configurable. Is there a way to configure the parameters stated above? It's working well, it's just that once the sun decides he's had enough, the tracking becomes jittery and intermittent at certain angles.

**Comment by sthalik** [comment]
It's not the right solution to use post-processing in order to fix point detection. You probably want to enable auto threshold.

**Reply by Filion91** [reply]
Just to be clear : I'm using Aruco markers for tracking. I'm not the most experienced, but I did notice what I thought was a major improvement in tracking when adjusting the settings I mentioned using my previous camera.

**Reply by sthalik** [reply]
Good catch, I didn't notice it was Aruco when replying originally.

Can you get me a short video from the camera feed that I could replay into opentrack?

Furthermore, you can improve image quality by lowering the framerate. The image gets brighter so you can decrease gain.

**Reply by Filion91** [reply]
Eheh, maybe I am being a bit too difficult, in reality there is no issue to troubleshoot, I was just asking if there any ways to configure the brightness/contrast/saturation options like on my other webcam; it's more a matter of optimization. If it's not possible with the PS3Eye for some reason, it's not a huge problem, I could probably just get a more powerful lamp in my room. Either way, thank you for your help and your work on this project!
